apps/benchmark/build.js

```diff
@@ -0,0 +1,40 @@
+const esbuild = require('esbuild');
+const ImportGlobPlugin = require('esbuild-plugin-import-glob').default;
+const AliasPlugin = require('esbuild-plugin-alias');
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+
+const shouldWatch = yargs.argv.watch;
+
+const outDir = path.resolve(__dirname, '../../dist/apps/benchmark/');
+
+esbuild
+  .build({
+    entryPoints: ['./src/index.tsx'],
+    outfile: path.join(outDir, 'bundle.js'),
+    minify: true,
+    bundle: true,
+    plugins: [
+      AliasPlugin({
+        '@griffel/core': path.resolve(__dirname, '../../packages/core/src/index.ts'),
+        '@griffel/react': path.resolve(__dirname, '../../packages/react/src/index.ts'),
+      }),
+      ImportGlobPlugin(),
+    ],
+    ...(shouldWatch && {
+      watch: {
+        onRebuild(error, result) {
+          if (error) console.error('watch build failed:', error);
+          else console.log('watch build succeeded:', result);
```

Comment:
```suggestion
          if (error) {
            console.error('watch build failed:', error);
          } else {
            console.log('watch build succeeded:', result);
          }  
```

_nit:_ We don't need to save lines